src/metadata/routes/schemas/avus.clj

```diff
@@ -49,3 +49,8 @@
 (s/defschema FilterByAvusRequest
   (merge TargetFilterRequest
          {:avus (describe [AvuFilterRequest] "The AVUs to use to filter the given targets")}))
+
+(s/defschema DeleteTargetAvusRequest
+  (merge TargetIDList
+         TargetTypesList
+         {:avus (describe [(select-keys Avu [:attr :value :unit])] "The AVUs to match for removal")}))
```

Comment:
Wasn't sure if `Avu` should be title case here
```suggestion
         {:avus (describe [(select-keys avu [:attr :value :unit])] "The AVUs to match for removal")}))
```

Reply:
Ah, yes, here it should be title case since it's referring to the `Avu` schema defined earlier in the file: https://github.com/cyverse-de/metadata/blob/bcbb1aea4d1758eb7547e5db293d725af615bf55/src/metadata/routes/schemas/avus.clj#L13

Reply:
Ah, I was poking around for an occurrence and missed that :p